changelog.md

```diff
@@ -1,12 +1,31 @@
 ## Changes from 0.10.0 to 0.11.0
 
-### (Potentially) Breaking Changes
+### Breaking Changes
+
+* Java 8 or higher is needed to run Marathon, since Java 6 and 7 support has reached end of life. 
 
 * `--revive_offers_for_new_apps` is now the default. If you want to avoid resetting filters
   if new tasks need to be started, you can disable this by `--disable_revive_offers_for_new_apps`.
 
 ### Overview
 
+#### Marathon uses Java 8 
+
+Java 8 is out for over a year now. The support for older Java versions have reached end of life. 
```

Comment:
Java 8 has been out for more than a year now. The support for older versions of Java has reached end of life.